### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by ProPontifex**
See title

### CODE REVIEW

**Review by Alstruit — DISMISSED** [review]

File: streams/us.m3u [hunk 1]
```diff
@@ -341,8 +337,6 @@ https://cbsta49f-cbsta49f-ms.global.ssl.fastly.net/amagi7b98-AmagiMixible/master
 https://eyeonesports.com/ES2RA-628g.m3u8
 #EXTINF:-1 tvg-id="EternalLifeTVNetwork.us",Eternal Life TV Network (1080p)
 https://livecdn.live247stream.com/eternallife/tv/playlist.m3u8
-#EXTINF:-1 tvg-id="",Everyday Refresh
-https://qvc-amd-live.akamaized.net/hls/live/2034113/lsqvc6us/master.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:345]: works for me

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: streams/us.m3u (hunk 1 above)
Comment (BellezaEmporium) [reply streams/us.m3u:345]: Can confirm it works for me too.

File: streams/us.m3u [hunk 2]
```diff
@@ -587,8 +579,6 @@ https://fl5.moveonjoy.com/MTV_2/index.m3u8
 https://fl5.moveonjoy.com/MTV_CLASSIC/index.m3u8
 #EXTINF:-1 tvg-id="MTVLive.us",MTV Live (720p)
 https://fl5.moveonjoy.com/MTV_LIVE/index.m3u8
-#EXTINF:-1 tvg-id="",mtvU (480p)
-https://fl5.moveonjoy.com/MTV_U/index.m3u8
```
Comment (BellezaEmporium) [inline streams/us.m3u:591]: http://fl5.moveonjoy.com/MTV_U/index.m3u8 works

**Comment by Dum4G** [comment]
@ProPontifex please consider to follow request changes

**Comment by ProPontifex** [comment]
Why was this closed?

**Comment by BellezaEmporium** [comment]
Seems like it's linked to what @Dum4G posted.

**Comment by ProPontifex** [comment]
Um, ok

**Comment by Dum4G** [comment]
I have nothing against reopening this case but please consider not removing channels that work

**Comment by ProPontifex** [comment]
Same here, but if I removed the channels they weren't working on my end (in Canada). If you point out specific examples, I'll be glad to reconsider.

**Pull request closed without merging**